js/feedSettings.js

```diff
@@ -57,11 +57,8 @@ $(function() {
 	var clipboard = new Clipboard('#rssurl .icon-clippy');
 	clipboard.on('success', function(e) {
 		$input = $(e.trigger);
-		$input.tooltip({placement: 'top', trigger: 'manual', title: t('core', 'Copied!')});
-		$input.tooltip('show');
-		_.delay(function() {
-			$input.tooltip('hide');
-		}, 3000);
+		$input.attr('title', 'show');
```

Comment:
I don't think this is intended?
![grafik](https://user-images.githubusercontent.com/213943/212764048-db27d19d-9f89-4f59-9b84-d2e809e3a4f7.png)

```suggestion
```